indico/htdocs/js/indico/modules/contributions/common.js

```diff
@@ -236,6 +236,50 @@
             var target = $(evt.target);
             reloadManagementAttachmentInfoColumn(target.data('locator'), target.closest('td'));
         });
+
+
+        $('#subcontribution-list table').sortable({
+            items: '.js-sortable-subcontribution-row',
+            handle: '.ui-sortable-handle',
+            placeholder: 'sortable-placeholder',
+            tolerance: 'pointer',
+            distance: 10,
+            axis: 'y',
+            containment: '#subcontribution-list table',
+            start: function(e, ui){
+                ui.placeholder.height(ui.helper.outerHeight());
+            },
+            update: function(e, ui) {
+                var self = $(this);
+                var oldIndex = ui.item.data('position');
+                var newIndex = ui.item.index() + 1;
+
+                if (oldIndex === newIndex) {
+                    return;
+                }
+
+                var subcontributionIds = self.sortable('toArray');
+                var positions = {};
+
+                ui.item.data('position', newIndex);
+
+                $.map(subcontributionIds, function(subcontributionId) {
+                    var row = self.find('#' + subcontributionId);
+                    var newIndex = row.index() + 1;
+
+                    positions[subcontributionId] = newIndex;
+                    row.find('.subcontribution-position').text(newIndex);
+                });
+
+                $.ajax({
+                    url: ui.item.data('sort-url'),
+                    method: 'POST',
+                    data: {subcontribution_id: subcontributionIds, positions: JSON.stringify(positions)},
+                    complete: IndicoUI.Dialogs.Util.progress(),
```

Comment:
i think this is one of the cases where it might look better without a progress dialog